### DISCUSSION TITLE
✅ 🔗 Update Icon Twitter into X (Duplicate of #23345)

### DISCUSSION THREAD

**Discussion by ghost** [elementor/elementor#24110] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When adding social share or social follow icons, the logo of Twitter is still not updated to X after such long period.

### Describe the solution you'd like

Change the icon to the X logo please in the next update.

### Describe alternatives you've considered

I added a custom icon, but I can't use these for social share buttons.

**Comment by nicholaszein** [comment]
Hello there, @larsnvg!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #23345

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hey @larsnvg!

### As you might already know, the 𝕏 logo for Twitter was implemented with the release of Elementor and Elementor Pro version 3.20.0. 🙌 
You can now use the Icon Library to include the 𝕏 logo on any icon location on your website. 

### Social Icons
For the [Social Icons widget](https://elementor.com/help/social-icons-widget/), you can use the Icon Picker and find the 𝕏 icon in the icon library by typing either **_twitter_** or **`x `** (with a space) in the search box. 

**It's important to note that this icon is only available if you have <ins>purged Object Cache</ins>. Otherwise you won't see the icon in the library!**

### Share Buttons (Pro)
For the [Share Buttons widget (Pro)](https://elementor.com/help/share-buttons-widget-pro/), we used a different approach. To maintain Backward Compatibility we added a new Network called **X** to the Network Dropdown List. You need to manually select that option to see the new icon.

### Backward Compatibility
Elementor takes backward compatibility very serious. We have no intention to make changes to existing features that may break your site. For this reason we decided not to replace all icons with the new 𝕏 icon right away. This ensures all existing sites remain intact, and gives you the option to chose if you want to use the new icon or not.

----

Thank you everyone for your feedback and suggestions! 🙏 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
